features/quizmaster/running_quiz.feature

```diff
@@ -20,3 +20,17 @@ Scenario: I send the first question
   When I press the "Send" button for question "What is 2+2?"
   And I switch to window "1"
   Then I should see "What is 2+2?"
+
+Scenario: I send multiple questions
+  Given I have sent the first question
+  And I click the "Correct" button
+  When I click the "Back to Questions" link
+  And I wait for the page to load
+  And I should see "Correct"
+
+Scenario: Resetting the quiz
+  Given I have sent the first question
+  And "Craft Academy" has answered question "What is 2+2?" with "Six"
+  When I press the "Reset the Quiz" button
+  Then I should not see "Correct"
+  And there should be "0" answers for "What is 2+2?"
```

Comment:
If you write this as `And there should be 0 answers for "What is 2+2?"` Cucumber will extract the number with a regex giving you a step definition that looks like

```gherkin
Then(/^there should be (\d+) answers for "([^"]*)"$/) do |count, question_body|
  question = Question.find_by(body: question_body)
  expect(question.answers.count).to eq count
end
```

note that I don't need to do `count.to_i` since it's already an integer value and not a string